geometry/SettingsViewController.swift

```diff
@@ -8,20 +8,18 @@
 
 import UIKit
 
-protocol SettingsViewControllerDelegate: class {
-    func settingsViewControllerDelegate(sender: SettingsViewController)
+protocol SettingsViewControllerDelegate {
+    func fillSettingsViewControllerDelegate(rColor: Float, gColor: Float, bColor: Float)
 }
 
 class SettingsViewController: UIViewController
 {
-    var settings = Settings()
+    //var settings = Settings()
     
-    weak var delegate: SettingsViewControllerDelegate?
-    
-    @IBAction func close(sender: AnyObject) {
-        dismissViewControllerAnimated(true, completion: nil)
-        delegate?.settingsViewControllerDelegate(self)
-    }
+    var delegate: SettingsViewControllerDelegate?
+    var redsvc = Float()
+    var greensvc = Float()
+    var bluesvc = Float()
     
     @IBOutlet weak var redСolor: UISlider!
```

Comment:
redColor should be renamed to redSlider. 'red' is already a color so it's redundant notation.